.github/ISSUE_TEMPLATE/new-dbt-test.md

```diff
@@ -0,0 +1,49 @@
+# New DBT Test
```

Comment:
**[Nitpick]** As you can see from [the official website](https://docs.getdbt.com/), the official style is actually all lowercase for dbt:

```suggestion
# New dbt test
```